dbcon/execplan/calpontsystemcatalog.cpp

```diff
@@ -3546,13 +3554,13 @@ const CalpontSystemCatalog::ROPair CalpontSystemCatalog::tableRID(const TableNam
   oss << "select objectid from systable where schema='" << aTableName.schema << "' and tablename='"
       << aTableName.table << "' --tableRID/";
 
-  csep.data(oss.str());  //@bug 6078. Log the statement
-
   if (fIdentity == EC)
     oss << "EC";
   else
     oss << "FE";
 
+  csep.data(oss.str());  //@bug 6078. Log the statement
```

Comment:
It might change the output. Did it break any tests?

Reply:
No this change did not break any tests in MTR or test001.